spec/features/admin_categories_view_spec.rb

```diff
@@ -28,7 +27,7 @@
     end
 
     it "can edit a category" do
-      within('#category_vegan') do 
+      within('#category_vegan') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.